tests/acceptance/DatasetUpdatesTest.php

```diff
@@ -27,16 +27,7 @@ public static function provideValidNumbers()
         return array(
             array('(11) 99129-2344'),
             array('(11) 93013-6165'),
-            array('(96) 9129-2344'), // AP
-            array('(96) 9129-2344'), // AP
-            array('(92) 9129-2344'), // AM
-            array('(97) 3412-1234'), // AM
-            array('(98) 9129-2344'), // MA
-            array('(99) 9129-2344'), // MA
-            array('(91) 9129-2344'), // PA
-            array('(93) 9129-2344'), // PA
-            array('(94) 9129-2344'), // PA
-            array('(95) 9129-2344'), // RO
```

Comment:
I think those phones should all have an additional 9 in front of them, right? Testing for that instead of removing them should be better.